src/index.js

```diff
@@ -1,9 +1,15 @@
 import { parse } from "babylon-lightscript";
 import { defaultImports, lightscriptImports, lodashImports } from "./stdlib";
 
+import { arrayStatementVisitor, objectStatementVisitor } from "./for-in";
+
 export default function (babel) {
   const { types: t } = babel;
 
+  // XXX: don't love this, but scrolling through a 2000 line file is maddening.
+  const forInArrayStatementVisitor = arrayStatementVisitor(babel);
```

Comment:
Well it is, as they say, your show -- but I think it's really bad from a maintainability standpoint to have that much code in one file.